nerdlandbot/commands/recipe.py

```diff
@@ -0,0 +1,153 @@
+            embed = discord.Embed(
+                title = embed_title,
+                description = questions[i],
+                color = NOTIFY_EMBED_COLOR
+            )
+            await ctx.send(embed=embed)
+            try:
+                reaction = await ctx.bot.wait_for("message", timeout=30, check=check(ctx.author))
+                await asyncio.sleep(1)
+                answers.append(reaction)
+                # If the user wants to abort, he can enter '0'
+                if reaction.content == "0":
+                    abort = translate("recipe_abort", lang)
+                    embed = discord.Embed(
+                        title = embed_title,
+                        description = abort,
+                        color = NOTIFY_EMBED_COLOR
+                    )
+                    return await ctx.send(embed=embed)
+                # When the user is asked a rating, check if it's a number between 1 and 5
+                if i == 2:
+                    if reaction.content.isnumeric():
+                        reaction_int = int(reaction.content)
+                        if reaction_int > 5 or reaction_int < 1:
+                            rating_error = translate("recipe_rating_error", lang)
+                            embed = discord.Embed(
+                                title = embed_title,
+                                description = rating_error,
+                                color = NOTIFY_EMBED_COLOR
+                            )
+                            return await ctx.send(embed=embed)
+                    else:
+                        int_error = translate("recipe_int_error", lang)
+                        embed = discord.Embed(
+                            title = embed_title,
+                            description = int_error,
+                            color = NOTIFY_EMBED_COLOR
+                        )
+                        return await ctx.send(embed=embed)
+            except asyncio.TimeoutError:
+                timeout = translate("recipe_timeout", lang)
+                embed = discord.Embed(
+                    title = embed_title,
+                    description = timeout,
+                    color = NOTIFY_EMBED_COLOR
+                )
+                return await ctx.send(embed=embed)
+        
+        # Let the user know the process has completed and he/she/it just has to wait now
+        processing = translate("recipe_processing", lang)
+        embed = discord.Embed(
+                    title = embed_title,
+                    description = processing,
+                    color = NOTIFY_EMBED_COLOR
+                )
+        msg = await ctx.send(embed=embed)
+
+
+        # Updating the worksheet(ws) with all the data asked of the user
+        ws.update("A{}".format(next_row), date_string)
+        ws.format("A{}".format(next_row), {"horizontalAlignment": "RIGHT"})
```

Comment:
Very minor comment: the timestamp is aligned left in the sheet, so I think this is not necessary? (not stopping approval for this)